examples/Tweet.hs

```diff
@@ -108,7 +97,10 @@ main = runBH' $ do
   let boost = Nothing
```

Comment:
Hallo Yannick,
I run into the same problems yesterday.
Missed your PR before I did my own identical [PR #204].
I solved the putTemplate with [PR #205 ]

Cheers
/Lars